src/containers/MasterWindow.js

```diff
@@ -22,7 +22,7 @@ class MasterWindow extends Component {
         }
     }
 
-    closeModalCallback = (isNew) => {
+    closeModalCallback = (type, isNew) => {
```

Comment:
I am bit confused, why we are declaring variable that is never used?

So i checked, and i see that closeCallback for modal has a lot useless parameters, shouldn't we get rid of them instead?